brainstormforce/ultimate-addons-for-gutenberg
# Improvement: FAQ Schema - Revamped the Padding options.

### Description
Trello task - https://trello.com/c/77TIcgei/146-faq-add-common-padding-option-for-the-whole-box-and-bottom-padding-option-for-question

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
 - Check Padding options.
 - Check Backward Compatibility 

### Checklist:
